deployments/operator/device/qat/qat.yaml

```diff
@@ -5,6 +5,13 @@ metadata:
   namespace: inteldeviceplugins-system
 spec:
   template:
+    # example apparmor annotation
```

Comment:
just realized these are in a wrong place. they belong to `deployments/operator/samples/`

Reply:
fixed